plugins/UsersManager/API.php

```diff
@@ -988,6 +997,13 @@ public function deleteUser($userLogin)
         $this->model->deleteUserOptions($userLogin);
         $this->model->deleteUserAccess($userLogin);
 
+        $container = StaticContainer::getContainer();
+        $email = $container->make(UserDeletedEmail::class, array(
+            'login' => Piwik::getCurrentUserLogin(),
+            'emailAddress' => Piwik::getCurrentUserEmail()
+        ));
+        $email->send();
+
```

Comment:
Just out of curiosity. Does it really make sense to email the current user that he just created/deleted a user?
Image someone wants to set up a new Matomo for a bigger company and creates a large number of accounts using the API. He would then be spammed with emails...
Imho it would be more helpful if maybe the superuser, or any defined user would be informed if any user creates/removes a user

Reply:
Yes it be great if the user got notified. That's because you want to know if someone else created a user using your user in case your login got hacked for example. A superuser might not be able to know if that was on purpose or not. And the emails you receive will be also be useful from an audit log perspective etc. 

Additionally maybe a superuser etc would want to be notified which I think we will be implementing using ActivityLog to be notified by email about certain events.

When many users are created regularly using the API then this might not be wanted but then we could eventually still have settings to disable certain emails (or a group of emails) but that be likely not needed for now and we could add it when needed.